src/extension.ts

```diff
@@ -11,9 +11,13 @@ function rangeWholeFile(doc: vscode.TextDocument): vscode.Range {
 }
 
 function getFormattedString(doc: vscode.TextDocument): string {
+	const documentUri = vscode.Uri.parse(doc.fileName);
```

Comment:
Could we use `doc.uri` directly instead?
I couldn't parse the file name as a URI correctly on Windows (drive letters like "C:" was treated as a scheme).

Reply:
fixed